application/console/controllers/CronController.php

```diff
@@ -116,4 +116,46 @@ public function actionDeleteInactiveUsers()
     {
         User::deleteInactiveUsers();
     }
+
+    /**
+     * Run all cron tasks, catching and logging errors to allow remaining tasks to run after an exception
+     */
+    public function actionAll()
```

Comment:
Since we're changing what cron commands need to be in the skinny repos that use ID Broker, it seems like this might be a major version bump.

Reply:
Oh, never mind. I see that this is merely adding a new way, not yet removing the old way these commands could be called.

It's probably worth documenting that the old way is deprecated (if we plan to remove it in some future version).

Reply:
sure, I don't know if the actual "api" changes, just how we're invoking it, but it's not a problem to do a major version change